component/src/governance/component.rs

```diff
@@ -33,23 +33,13 @@ impl Component for Governance {
             .expect("enacting proposals should never fail");
 
         // TODO: This will need to be altered to support dynamic epochs
```

Comment:
did this comment refer to the if statement ("need to alter the condition"), or the contents of `end_epoch`? in either case i think it's dangling now